### DISCUSSION TITLE
Invalid workflow file error

### DISCUSSION THREAD

**Discussion by brlin-tw** [community/community#26699] [Actions]
<p>I need help on why this workflow file doesn’t work: <a href="https://github.com/the-common/bash-script-templates/actions/runs/2235308036/workflow" rel="noopener nofollow ugc">https://github.com/the-common/bash-script-templates/actions/runs/2235308036/workflow</a>, which I only modified some <code>name</code>s using a known good template.</p>
<p>The <code>The workflow is not valid.</code> error message isn’t helpful either.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p><code>files</code> is not a valid step attribute. If <code>files</code> is supposed to be an input parameter for the action you’re using in that step, it needs to go into the <a href="https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepswith"><code>with</code> attribute of that step</a>, like so:</p>
<pre><code class="lang-yaml">      - name: Deliver build products to GitHub Releases
        uses: softprops/action-gh-release@v0.1.13
        with:
          files: |
            ${{ env.release_id }}.tar*
</code></pre>
